docker-compose/_posts/2017-06-08-adding-ssh-keys.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Adding SSH Keys
+short: Adding SSH Keys
+category: docker-compose
+order: 4
+development: true
+---
+
+Adding your SSH key allows services in your project to access any private repositories they depend on when building.
```

Comment:
Would reword this so it doesn't repeat the heading:
<img width="769" alt="screen shot 2017-06-12 at 11 44 53 am" src="https://user-images.githubusercontent.com/5341618/27049574-96c6812a-4f64-11e7-9362-3dae3fff48d4.png">

Maybe something like:

> Grant your projects access to private repositories during its build.